src/test/basics/base_uint_test.cpp

```diff
@@ -68,15 +93,15 @@ struct base_uint_test : beast::unit_test::suite
         {
             BEAST_EXPECT(d == --t);
         }
```

Comment:
I see what you are saying - test code is a little different than core code and I think the linearity here is actually a benefit to understanding what is being tested with the expected state of each object expressed clearly by each statement. Wrapping this up in a method to save a little typing seems like it would actually decrease the readability of the test. That said, if you feel really strongly about the change, feel free to say so and I'll try to implement your suggestion. Thanks.